bin/alluxio

```diff
@@ -217,13 +217,10 @@ function main {
           # if ufs is local filesystem and already exists
           exit 0
         else
-          declare -a schemes=(hdfs s3 s3a s3n glusterfs swift oss gcs)
-          for scheme in ${schemes[@]}; do
-            if [[ ${ALLUXIO_UNDERFS_ADDRESS} == ${scheme}://* ]]; then
-              # if ufs is not local filesystem, don't format
-              exit 0
-            fi
-          done
+          if [ ! -z ${ALLUXIO_UNDERFS_ADDRESS} ] && [[ ${ALLUXIO_UNDERFS_ADDRESS} != /* ]] && [[ ${ALLUXIO_UNDERFS_ADDRESS} != file://* ]]; then
```

Comment:
is the `[! -z ${ALLUXIO_UNDERFS_ADDRESS}]` needed when we also check `[ ${ALLUXIO_UNDERFS_ADDRESS} != /*`?

Reply:
@aaudiber it is. if `${ALLUXIO_UNDERFS_ADDRESS}` is empty all not conditions will be true and it will exit (assume no local ufs)

Reply:
If the UFS address is empty, shouldn't we skip formatting because it isn't a local UFS?

Reply:
@aaudiber I believe the following `alluxio.cli.Format` class will default to local fs for the journal

Reply:
I see, sgtm then. Can you add a comment that empty string results in using the default UFS value, which is a local filesystem?

I think we should also use `[[ ... ]]` instead of `[ ... ]`

Reply:
@aaudiber done